packages/react-events/src/Focus.js

```diff
@@ -249,6 +305,7 @@ const FocusResponder = {
           nativeEvent.key === 'Tab' &&
           !(nativeEvent.metaKey || nativeEvent.altKey || nativeEvent.ctrlKey)
         ) {
```

Comment:
Im wondering if this is a correct condition. MacOS (and therefore Safari) wont focus i.e. radios and checkboxes on Tab alone. U need to press down Tab+Alt for that